debugger/src/debugger.rs

```diff
@@ -322,7 +330,7 @@ pub fn start_session(debugger_options: &DebuggerOptions) -> Result<SessionData,
         .mode(ArchMode::Thumb)
         .endian(Endian::Little)
         .build()
-        .unwrap();
+        .map_err(|err| anyhow!("Error creating Capstone disassembler: {:?}", err))?;
```

Comment:
The recommended way for this is to use the `.with_context` function, as described in the [anyhow Documentation].

[anyhow Documentation]: https://docs.rs/anyhow/1.0.45/anyhow/trait.Context.html#tymethod.with_context

Reply:
I tried this ...
```
.with_context(|err| "Error creating Capstone disassembler.")?;
```
and got an error which I confess to not fully understanding. Google searches didn't enlighten me much either.  @Tiwalun Can you help?
```
the method `with_context` exists for enum `Result<Capstone, capstone::Error>`, but its trait bounds were not satisfied
the following trait bounds were not satisfied:
`capstone::Error: anyhow::context::ext::StdError`
which is required by `Result<Capstone, capstone::Error>: anyhow::Context<Capstone, capstone::Error>`rustcE0599
result.rs(503, 1): doesn't satisfy `_: anyhow::Context<Capstone, capstone::Error>`
error.rs(13, 9): doesn't satisfy `capstone::Error: anyhow::context::ext::StdError`
```

Reply:
It seems that `capstone::Error` does not implement `std::error::Error`, and because of this it's not compatible with the `with_context` method. Creating a new error, like you did, is probably the best solution in that case.